Sources/SwiftUIKit/Navigation/Navigate.swift

```diff
@@ -224,7 +224,7 @@ public class Navigate {
                         $0.backgroundColor = style.color
                         $0.clipsToBounds = true
                 }
-                .layer { $0.cornerRadius = 8 }
+                .layer(cornerRadius: 8)
```

Comment:
magic number, should this be a default somewhere?

Reply:
This is a preference... I wouldn't consider it a magic number... simply I base most spacing/ numbers off `4` or `8`.